packages/app/src/components/nominationInfo/applicationDetail.test.js

```diff
@@ -3,38 +3,42 @@ import { render } from "@testing-library/react";
 import ApplicationDetail from './applicationDetail';
 import dummyData from './dummyData';
 
-test("renders prop name providerName", () => {
-  const fields = [
+test("renders prop name patientInfo", () => {
+
+  const diffDays = Math.round(Math.abs((dummyData.admissionDate
+    - dummyData.dischargeDate) / (24*60*60*1000))) >= 21 ? 'Yes' : 'No';  /* <- hours*minutes*seconds*milliseconds */
```

Comment:
This logic seems a little too complex for a test. Since the dummy data were using here is static, I think it's fine just hard-code `diffDays`. Generally-speaking you want to avoid dynamically-genrated test data to keep your tests predictable.